build/start_mariadb.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
```

Comment:
add unset proxy setting here: 
```
+ # Unset proxy environment variables to avoid interference
+ unset http_proxy HTTP_PROXY
+ unset https_proxy HTTPS_PROXY
+ unset no_proxy NO_PROXY
+ unset all_proxy ALL_PROXY
+ unset ftp_proxy FTP_PROXY
```